.editorconfig

```diff
@@ -0,0 +1,6 @@
+
+root = true
+
+[*.{odin,c,h,cpp}]
```

Comment:
And `py`, for the big math test rig.

Reply:
Maybe I should just say tabs to all files? To not bother with extensions... I don't think anything would break with tabs